components/google_calendar/google_calendar.app.js

```diff
@@ -1,154 +1,307 @@
-const { google } = require("googleapis")
-const get = require("lodash.get")
+const { google } = require("googleapis");
```

Comment:
Optional:

It would be great if you can migrate the whole component to ES Modules here you can [see the docs](https://pipedream.com/docs/components/guidelines/#components-should-be-es-modules) on how to do it